src/types/common.types.ts

```diff
@@ -12,16 +12,12 @@ type _HTTPMethods = 'DELETE' | 'GET' | 'HEAD' | 'PATCH' | 'POST' | 'PUT' | 'OPTI
 
 export type HTTPMethods = Uppercase<_HTTPMethods> | Lowercase<_HTTPMethods>
 
-export type ApiRequestProps = {
+export type RequestOption = {
         method: HTTPMethods,
-        url: string,
         headers?: GenericObjectType,
         body?: unknown,
```

Comment:
will this still be body or data ? @Amitkanswal

Reply:
it's body in app-sdk as we want to make it close to fetch. We are mapping it to data in AppExtensionComponent. To work with proxy service